custom/icds_reports/utils/aggregation_helpers/distributed/ccs_record_monthly.py

```diff
@@ -61,10 +62,13 @@ def aggregation_query(self):
         ).format(end_month_string, start_month_string)
 
         alive_in_month = "(case_list.date_death is null OR case_list.date_death-{}>0)".format(start_month_string)
-        migration_status = "(agg_migration.is_migrated=1 AND agg_migration.migration_date < {})::integer".format(
-            start_month_string)
-        seeking_services = "(person_cases.registered_status IS DISTINCT FROM 0 AND {} IS DISTINCT FROM 1)".format(
-            migration_status)
+        migration_status = "(agg_migration.is_migrated = 1 AND agg_migration.migration_date::date < {" \
+                           "start_month_string})::integer".format(start_month_string=start_month_string)
```

Comment:
my previous comments about coercion to integer still hold (its a bit hard to tell because of the rebases)

Reply:
Updated!